build.gradle

```diff
@@ -45,27 +57,33 @@ subprojects {
     targetCompatibility = 1.7;
 
     repositories {
-        mavenCentral()
+        //mavenCentral()
```

Comment:
Required either commenting (e.g. why) or removal.

I also suspect this may have to be reverted back.